docs/source/configuration.rst

```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
 route of ``GetEntityGenericRoot`` if that exists. If not, it uses the
 the reverse route of ``apis_core:GetEntityGeneric``, which is defined in
 :py:mod:`apis_core.urls` and defaults to ``/entity/{pk}``.
+
+Django Settings
+---------------
+
+This section deals with the internal configuration of the APIS tool. For instructions on how to set it up please refer
+to :doc:`installation`.
+All the settings described here are part of the Django settings and can be set in the ``settings.py`` file of your Django project.
+
+
+REST_FRAMEWORK
+^^^^^^^^^^^^^^
+
+APIS uses the `Django Restframework`_ for API provisioning. Restframework specific settings like the default page size can be set here.
+
+.. code-block:: python
+
+    REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"] = (
+        "rest_framework.permissions.DjangoObjectPermissions",
+    )
+
+Use the above default to allow setting permissions on object level. Meaning that every user gets his permissions depending on his/her user group and the collections this group has permissions for.
```

Comment:
we don't use django guardian anymore, so we don't have objectlevel permissions, do we?

Reply:
goof point, no we dont

Reply:
removed that and kept ModelPermission